rmf_site_editor/src/wireframe.rs

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+use crate::menu_bar::{MenuEvent, MenuItem, ViewMenu};
+use bevy::pbr::wireframe::{Wireframe, WireframePlugin};
+use bevy::prelude::*;
+
+use rmf_site_format::ModelMarker;
+
+#[derive(Default)]
+pub struct SiteWireframePlugin;
+
+/// Keeps track of which entity is associated to the toggle menu button
+#[derive(Resource)]
+pub struct WireframeMenu {
+    toggle_wireframe: Entity,
+}
+
+impl FromWorld for WireframeMenu {
+    fn from_world(world: &mut World) -> Self {
+        // Tools menu
+        let toggle_wireframe = world
+            .spawn(MenuItem::CheckBox("View wireframe".to_string(), false))
```

Comment:
Nit: To match the other view options
```suggestion
            .spawn(MenuItem::CheckBox("Wireframe".to_string(), false))
```